src/scrapeLinks.test.js

```diff
@@ -1,47 +1,61 @@
 import scrapeLinks from "./scrapeLinks";
 
-const testMarkdownString = `
-This is a Markdown example with [a link to google](https://www.google.com) and [one with a subdirectory](https://www.google.com/nested/page.html)
+const markdownString = `
+Markdown sample with a [link to Google](https://www.google.com), one [with a URL path](https://www.google.com/nested/page.html), and others:
 
-and [another to reddit](www.reddit.com) and [a third to Twitter](facebook.com)
+- One [to reddit](www.reddit.com)
+- A fourth [to Facebook](facebook.com) (incomplete URLs)
+- Finally a few [ref] [links][links] [here][link-here]
+- BTW it shouldn't plain links like http://www.this.com or that.com shouldn't get picked up.
```

Comment:
I added this line in `markdownString` but actually I'm not sure. Do these get turned to actual links in https://github.com/iterative/dvc.org ?

Reply:
No idea, I've never used these ref links before. Wasn't there a test PR that included these?

Reply:
Here I was only referring to the last line with the plain links. BTW I quickly tested this and MD does link some of these so... The main regex may nee more tweaks? I can open a separate issue ⌛ 

![image](https://user-images.githubusercontent.com/1477535/123993305-ee834e00-d991-11eb-8494-9f8b2a3ce3ce.png)

![image](https://user-images.githubusercontent.com/1477535/123993353-fb07a680-d991-11eb-995e-e19f5ce0a44a.png)

Reply:
Moving this to https://github.com/iterative/link-check/issues/7. Resolving